packages/flutter/lib/src/gestures/team.dart

```diff
@@ -105,11 +110,24 @@ class _CombiningGestureArenaMember extends GestureArenaMember {
 /// the horizontal nor vertical drag recognizers can claim victory) the tap
 /// recognizer still actually wins, despite being in the team.
 ///
+/// [AndroidView] is using a team with a captain to decide which gestures are
+/// forwarded to the native view. For example if we want to forward taps and
+/// vertical scrolls to a native Android view, [TapGestureRecognizers] and
+/// [VerticalDragGestureRecognizer] are added to a team with a captain, once the
+/// captain wins then gesture is forwarded to the Android view.
```

Comment:
this is good as far as it goes but it doesn't answer the question of why it uses a captain.

Reply:
Tried to thicken the explanation a bit.